tree/dataframe/inc/ROOT/RDF/RColumnRegister.hxx

```diff
@@ -56,7 +56,8 @@ class RColumnRegister {
    std::shared_ptr<const std::unordered_map<std::string, std::string>> fAliases;
    /// Immutable multimap of Variations, can be shared among several nodes.
    /// The key is the name of an existing column, the values are all variations that affect that column.
-   /// As a consequence, Variations that affect multiple columns are inserted multiple times, once per column.
+   /// Variations that affect multiple columns are inserted in the map multiple times, once per column,
+   /// and conversely each column (i.e. each key) can have several associated variations.
```

Comment:
Why not reformulating the two sentences to: `When variations affect multiple columns, each varied column is inserted in the map`.

Reply:
I mean in the PR ;)

Reply:
It is all good.